etc/internal_options.conf

```diff
@@ -14,11 +14,11 @@
 # Analysisd default rule timeframe.
 analysisd.default_timeframe=360
 # Analysisd stats maximum diff.
-analysisd.stats_maxdiff=25000
+analysisd.stats_maxdiff=999000
```

Comment:
You have tested this?  I am just asking to make sure but it makes sense to me gives a lot more information to users.